pkg/kab/patch_manifest.go

```diff
@@ -33,6 +33,7 @@ const (
 	DOCKER_FOR_DESKTOP_NAME        = "docker-for-desktop"
 	NODE_PORT_ENV_VAR              = "NODE_PORT"
 	CNAB_INSTALLATION_NAME_ENV_VAR = "CNAB_INSTALLATION_NAME"
+	LABEL_KEY_NAME                 = "bundle-installation-name"
```

Comment:
Does this need qualifying so we don't accidentally delete resources created by others?

Reply:
@glyn I added the `bundle` prefix to qualify this label. I had also considered `cnab-k8s-installer-installation-name`, but chose the former. Would you prefer the later?

Reply:
Yes, I think that's safer.